modules/nw-sriov-example-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-example-dpdk-line-rate_{context}"]
+= Using SR-IOV and the Node Tuning Operator to achieve a DPDK line rate
+The Node Tuning Operator is used to configure isolated CPUs, hugepages, and a topology scheduler.
+You can then use the Node Tuning Operator with SR-IOV to achieve a specific Data Plane Development Kit (DPDK) line rate networking.
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You have logged in as a user with `cluster-admin` privileges.
+* You have deployed a standalone Node Tuning Operator.
+
+[NOTE]
+====
+In previous versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title}{product-version} 4.11, this functionality is part of the Node Tuning Operator.
```

Comment:
You could replace "OpenShift applications" with "`{product-title}` applications".